coffee-module/coffee-module-redis/src/test/java/hu/icellmobilsoft/coffee/module/redis/config/JedisPoolTest.java

```diff
@@ -0,0 +1,96 @@
+/*-
+ * #%L
+ * Coffee
+ * %%
+ * Copyright (C) 2020 - 2022 i-Cell Mobilsoft Zrt.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package hu.icellmobilsoft.coffee.module.redis.config;
+
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+
+import org.jboss.weld.junit5.EnableWeld;
+import org.jboss.weld.junit5.WeldInitiator;
+import org.jboss.weld.junit5.WeldJunit5Extension;
+import org.jboss.weld.junit5.WeldSetup;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import hu.icellmobilsoft.coffee.module.redis.annotation.RedisConnection;
+import redis.clients.jedis.JedisPool;
+
+/**
+ * Test for Redis pool config producer
+ * 
+ * @author imre.scheffer
+ * @since 1.11.0
+ *
+ */
+@EnableWeld
+@Tag("weld")
+@ExtendWith(WeldJunit5Extension.class)
+@DisplayName("Redis pool config tests")
+class JedisPoolTest {
+
+    static final String CONFIG_KEY = "test";
+    static final String CONFIG_KEY_YML = "yamlconfig";
+    static final String POOL_CONFIG_KEY_CUSTOM1 = "custom1";
+
+    @Inject
+    @RedisConnection(configKey = CONFIG_KEY)
+    private JedisPool jedisPool;
+
+    @Inject
+    @RedisConnection(configKey = CONFIG_KEY_YML)
+    private JedisPool ymlDefaultJedisPool;
+
+    @Inject
+    @RedisConnection(configKey = CONFIG_KEY_YML, poolConfigKey = POOL_CONFIG_KEY_CUSTOM1)
+    private JedisPool ymlCustom1PoolJedisPool;
+
+    @WeldSetup
+    public WeldInitiator weld = WeldInitiator.from(WeldInitiator.createWeld()
+            // beans.xml scan
+            .enableDiscovery())
+            // start request scope + build
+            .activate(RequestScoped.class).build();
+
```

Comment:
Ide is lehetne akár egy redis mock alkalmazás? vagy azt rosszul értettem hogy saját docker image tud indulni a teszthez?, mert így ez a teszt elbukik hogyha nincs redis nem?

Reply:
a coffeban nincsen jelenleg integracios tesztre hely, igy csak klasszikus junit teszteket lehet megoldani. De ami itt van az mind mukodik redis szerver nelkul